drivers/modem/gsm_ppp.c

```diff
@@ -618,10 +618,8 @@ static void gsm_finalize_connection(struct gsm_modem *gsm)
 					    "AT", &gsm->sem_response,
 					    GSM_CMD_AT_TIMEOUT);
 		if (ret < 0) {
-			LOG_ERR("modem setup returned %d, %s",
-				ret, "retrying...");
-			(void)gsm_work_reschedule(&gsm->gsm_configure_work,
-						K_SECONDS(1));
+			LOG_ERR("%s returned %d, %s", "AT", ret, "retrying...");
+			(void)gsm_work_reschedule(&gsm->gsm_configure_work, K_SECONDS(1));
```

Comment:
Should there be a define for the` K_SECONDS(1)`? Its used a lot

Reply:
fixed in new commit